code/script.js

```diff
@@ -0,0 +1,56 @@
+var flag = false;
+
+// displays numerous inputs (numbers) on the screen
+function displaynumber(n1) {
+  calculator.display.value = calculator.display.value + n1;
+}
+
+// testing 0 input, when 0 comes after / buttons are disabled and message is given
+function testZero(n2) {
+  if (flag == true && n2 == 0) {
+    var b = document.getElementsByClassName("btn");
```

Comment:
I don't know what is b

Reply:
b is specified now such as var rCalcButton I used this because it has same name as its class="rcalc-button"